src/cplus_plugin/gui/implementation_model_widget.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+"""
+Container widget for configuring the implementation widget.
+"""
+
+import os
+import typing
+
+from qgis.PyQt import QtCore, QtGui, QtWidgets
+
+from qgis.PyQt.uic import loadUiType
+
+from ..conf import settings_manager
+
+from .model_component_widget import (
+    ImplementationModelComponentWidget,
+    NcsComponentWidget,
+)
+
+from ..models.base import ImplementationModel, NcsPathway
+
+from ..utils import FileUtils
+
+
+WidgetUi, _ = loadUiType(
+    os.path.join(
+        os.path.dirname(__file__), "../ui/implementation_model_container_widget.ui"
+    )
+)
+
+
+class ImplementationModelContainerWidget(QtWidgets.QWidget, WidgetUi):
+    """Widget for configuring the implementation model."""
+
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.setupUi(self)
+
+        self._items_loaded = False
+
+        self.btn_add_one.setIcon(FileUtils.get_icon("play.svg"))
```

Comment:
QGIS theme icons might be a better option here eg. https://github.com/qgis/QGIS/blob/master/images/themes/default/mActionPlay.svg and https://github.com/qgis/QGIS/tree/master/images/themes/default/temporal_navigation

Reply:
This is now effected.